test/modules/fundingManager/bondingCurve/utils/mocks/FM_BC_Bancor_Redeeming_VirtualSupplyV1Mock.sol

```diff
@@ -23,6 +23,9 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Mock is
     // The FM_BC_Bancor_Redeeming_VirtualSupply_v1 is not abstract, so all the necessary functions are already implemented
     // The goal of this mock is to provide direct access to internal functions for testing purposes.
 
+    //--------------------------------------------------------------------------
```

Comment:
Missing space between `//` and `---`
(maybe just update the whole file if you are at it?)